MerlinAU.sh

```diff
@@ -3094,22 +3438,20 @@ Please manually update to version $minimum_supported_version or higher to use th
     ##---------------------------------------##
     ## Added by ExtremeFiretop [2023-Dec-09] ##
     ##---------------------------------------##
-    # Get the required space for the firmware download and extraction
-    required_space_kb="$(get_required_space "$release_link")"
-    if ! _HasRouterMoreThan256MBtotalRAM_ && [ "$required_space_kb" -gt 51200 ]
+    # Get the required memory for the firmware download and extraction
+    requiredRAM_kb="$(_GetRequiredRAM_KB_ "$release_link")"
+    if ! _HasRouterMoreThan256MBtotalRAM_ && [ "$requiredRAM_kb" -gt 51200 ]
     then
         if ! _ValidateUSBMountPoint_ "$FW_ZIP_BASE_DIR" 1
         then
-            Say "${REDct}**ERROR**${NOct}: A USB drive is required for the F/W update due to limited RAM."
-            "$inMenuMode" && _WaitForEnterKey_ "$mainMenuReturnPromptStr"
-            return 1
+	@@ -3106,10 +3121,10 @@ Please manually update to version $minimum_supported_version or higher to use th
```

Comment:
This afternoon, I resumed the review of this PR and found this line **3447**:
`@@ -3106,10 +3121,10 @@ Please manually update to version $minimum_supported_version or higher to use th`

It looks like it's the result of a previous conflict. Can you take a look and see if you can resolve the conflict?

Reply:
Wow great catch! Yes let me solve this now

Reply:
@Martinski4GitHub 

I found the commit that broke everything:
https://github.com/ExtremeFiretop/MerlinAutoUpdate-Router/commit/98207c5a6867a0a65b54f26e40d391d75347a53a

Seems like it was when I tried to sync up your changes from your PR for 1.0.11 into my 1.1.0 branch.
I'm trying to decide if it's easiest to just rollback that specific commit and re-do it, or just rollback the whole branch

Reply:
@Martinski4GitHub 

Rolled back the commit successfully.
Confirmed the changes after continued to apply correctly and nothing appears broken at this time.

Will simply re-summit that commit so your memory changes from your PR for 1.0.11 correctly shows up